examples/archive.html

```diff
@@ -72,6 +73,12 @@ <h3 id="offcanvasRightLabel">Images</h3>
     let input = document.getElementById('input');
     let responseText = document.getElementById('response');
     let imageContainer = document.getElementById('imgContainer');
+    let imageGroup = L.distortableCollection().addTo(map)
```

Comment:
Let's rename this "placedImagesCollection" and remove the `addTo(map)`!
```suggestion
    let placedImagesCollection = L.distortableCollection()
```